tests/testEmailVerification.php

```diff
@@ -102,85 +109,76 @@ function() {
 		$this->assertEquals( '__test_auth0_verify_email_page__', ob_get_clean() );
 	}
 
-	/**
-	 * Test AJAX email verification send.
-	 *
-	 * @runInSeparateProcess
-	 */
-	public function testResendVerificationEmail() {
+	public function testThatResendActionFailsWhenBadAjaxNonce() {
```

Comment:
This is just splitting tests out, not changing what's actually being tested. Also removes the mocked method in favor of a mocked HTTP response.